scripts/build/clean.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Cleans top-level build directories (for OSS and internal processes),
+ * dist subfolders under packages, and .js[.map] files in package directories.
+ */
+
+const {exec} = require('child_process');
+const fs = require('fs');
+
+const {sync: globSync} = require('glob');
+
+function main() {
+  removeDirectory('build');
+  removeDirectory('.rewrite-tmp');
+  removeFilesOfType('css');
+  removeFilesOfType('js');
+  removeFilesOfType('d.ts');
+  removeFilesOfType('map');
+}
+
+function removeDirectory(directory) {
+  exec(`rm -rf ${directory}`, (err) => {
```

Comment:
Do we care about Windows support? You might want to add `del-cli` back to the root `package.json` and use it here instead of executing a shell command.

Reply:
ah...definitely